apps/emqx/src/emqx_schema.erl

```diff
@@ -3385,3 +3412,18 @@ ensure_default_listener(Map, ListenerType) ->
 
 cert_file(_File, client) -> undefined;
 cert_file(File, server) -> iolist_to_binary(filename:join(["${EMQX_ETC_DIR}", "certs", File])).
+
+mqtt_converter(undefined, _Opts) ->
+    undefined;
+mqtt_converter(#{} = Mqtt, _Opts) ->
+    Multiplier = maps:get(<<"keepalive_multiplier">>, Mqtt, ?DEFAULT_MULTIPLIER),
+    Backoff = maps:get(<<"keepalive_backoff">>, Mqtt, ?DEFAULT_BACKOFF),
+    case Multiplier == Backoff * 2 of
```

Comment:
Maybe compare the result of round(V * 100) instead of == two floats

Reply:
And if round-100 Multiplier is not default, we should return Mqtt directly (user changed it), discard backoff

Reply:
done